src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -87,3 +91,14 @@ export const DropdownMenu = forwardRef<HTMLDivElement, DropdownMenuProps>(
     );
   }
 );
+
+export interface HeaderProps {
+  /** Item to render inside the Header */
+  item: ReactNode;
```

Comment:
Can we change `item` to `children`? This way we'll be able to use the component as `<DropdownHeader>Header Text</DropdownHeader>` - a little bit of a cleaner API. Tests will need to be updated for this